api/main.py

```diff
@@ -90,7 +90,7 @@ async def get_parallels_for_middle(parallels: ParallelsCollection):
     query_result = get_db().AQLQuery(
         query=main_queries.QUERY_PARALLELS_FOR_MIDDLE_TEXT,
         batchSize=10000,
-        bind_vars={
+        bindVars={
```

Comment:
Why did you change all these? CamelCase is for Javascript.

Reply:
this unfortunately is due to the nature of python arangodb bindings. They require the camelCasing, we can't really change it :(